src/llmcompressor/transformers/finetune/session_mixin.py

```diff
@@ -486,8 +486,11 @@ def save_model(self, output_dir: str, _internal_call=False, _is_oneshot=False):
             )
 
         self.save_state()
-        if self.tokenizer is not None:
-            self.tokenizer.save_pretrained(output_dir)
+        processor = self.processing_class if hasattr(self, "processing_class") else self.tokenizer
```

Comment:
```suggestion
        processor = getattr(self, "processing_class", self.tokenizer)
```

Reply:
I know I originally suggested the other way, but I think this is a little bit more concise